examples/dwm/bluetooth.lua

```diff
@@ -0,0 +1,141 @@
+-- A widget to display currently connected and paired bluetooth devices.
+-- To change output format modify reprint_devices function.
+
+separator = " "
+
+-- Object paths look like /org/bluez/hci0/dev_XX_XX_XX_XX_XX_XX/somethingsomething
+function get_device_mac_address(device_object_path)
+    return device_object_path:gsub("/.*/dev_", ""):gsub("/.*", ""):gsub("_", ":")
```

Comment:
What if `/somethingsometing` contains `/dev_`?

I would rewrite it as follows:
```lua
function get_device_mac_address(obj_path)
    local s = obj_path:match('/dev_([%x_]+)')
    return s and s:gsub('_', ':')
end
```

Reply:
Function won't encounter such string, probably, at least bluez documentation does not specify this. ![screen](https://user-images.githubusercontent.com/59440032/118862252-2a80b900-b8e6-11eb-8aa8-726344a666b9.png) (https://git.kernel.org/pub/scm/bluetooth/bluez.git/tree/doc)
Also object path returned by dbus plugin shouldn't contain `/somethingsomething` at all; stripping it is a leftover from earlier versions of a widget, where I tried to use different signals to detect devices connecting and disconnecting.